src/components/HappySlider/HappySlider.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import './HappySlider.scss'
+
+// Relative path does not work for css Variable
+// import images into a variable
+import smiley1 from './Smileys/smiley1.svg'
+import smiley2 from './Smileys/smiley2.svg'
+import smiley3 from './Smileys/smiley3.svg'
+import smiley4 from './Smileys/smiley4.svg'
+import smiley5 from './Smileys/smiley5.svg'
+
+function HappySlider(props) {
```

Comment:
Functional component! I meant to do this when I removed state from the original. Ooops.

Reply:
can you refactor to 
```js
export const HappySlider({ id, value, min, max, name, onChange }) => {
```
rather than `export default`
you can declare the constants here and eliminate line 17.
`import { HappySlider }` into other components

Reply:
Indeed. This is much nicer. Thanks.